node/manifest.ts

```diff
@@ -48,11 +53,26 @@ const generateManifest = ({
   bundles = [],
   declarations = [],
   functions,
+  functionConfig = {},
   importMap,
   layers = [],
 }: GenerateManifestOptions) => {
   const preCacheRoutes: Route[] = []
   const postCacheRoutes: Route[] = []
+  const manifestFunctionConfig: Manifest['function_config'] = {}
+
+  const getFunctionConfig = (name: string) => {
```

Comment:
I find this a bit confusing. We have a nested function that mutates an object in the outer scope but its name says it's a getter.

If I understand this correctly, the goal of this function is to prime `manifestFunctionConfig` with blank entries that you can safely mutate? Perhaps it would be nicer to make this function explicitly about that, by both changing the name and removing the return value so that it's explicitly a setter.

Then further below you could access `manifestFunctionConfig` directly rather than the return value of `getFunctionConfig`?

Reply:
Tried this out, and having a function called `primeFunctionConfig` also looked weird and added indirection. I tried a different approach where the config is filled up at the start, and then reduced before saving it: https://github.com/netlify/edge-bundler/pull/274/commits/520a241a0dc85cc6f58231cc973f75a9be5b452c How do you like that?